recipes/configs/code_llama2/7B_lora_single_device.yaml

```diff
@@ -0,0 +1,93 @@
+# Config for single device full finetuning in full_finetune_single_device.py
+# using a Code-Llama2 7B model
+#
+# This config assumes that you've run the following command before launching
+# this run:
+#     tune download codellama/CodeLlama-7b-hf --output-dir /tmp/CodeLlama-7b-hf
+#
+# The default config uses an optimizer from bitsandbytes. If you do not have it installed,
+# you can install it with
+#   pip install bitsandbytes
```

Comment:
Remove this, we don't use bitsandbytes in this config. Same comment for the QLoRA config too